documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM1.adoc

```diff
@@ -0,0 +1,26 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM1
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM1
+
+== New shortcuts in Fuse perspective
+
+Shortcuts for Java, Launch and Debug configuration and basic navigation are now enabled by default with Fuse perspective.
+
+The result is a set of actions now available in the Toolbar:
+
+image::./images/newToolbarAction.png[New Toolbar action]
+
+All the associated key shortcuts are also available, such as Ctrl+Shift+T to open a Java Type.
+
+== Performance improvement: loading Advanced tab for Camel components
+
+The load of Advanced tab in Properties view for Camel components has been greatly improved.
```

Comment:
"The loading time of the "Advanced" tab inside the Properties view of Camel Endpoints have been greatly improved."

Reply:
why have instead of has? the subject is singular "The loading time"

Reply:
you are right...my fault